app/src/styles/_colors.scss

```diff
@@ -29,16 +30,23 @@ $color-mapping: (
 	'success': $green,
 	'warning': $orange,
 	'danger': $red,
+	'gray': $gray,
 );
 
 // Suffix -> Percentage mapping (Percentage of main color)
 // Negative numbers will use secondary color for mixture.
 // The percentage will be equal to the absolute value of the number.
 $step-mapping: (
 	'10': 10,
+	'20': 20,
 	'25': 25,
+	'30': 30,
+	'40': 40,
 	'50': 50,
+	'60': 60,
+	'70': 70,
 	'75': 75,
+	'80': 80,
```

Comment:
This isn't really going to persist long enough.. These color palettes are scheduled to disappear, so lets try to find an alternative solution